spec.emu

```diff
@@ -0,0 +1,167 @@
+      </tbody>
+    </table>
+  </emu-table>
+</emu-clause>
+
+<emu-clause id="sec-properties-of-the-symbol-constructor">
+  <h1>Properties of the Symbol Constructor</h1>
+  <ins class=block><emu-clause id="sec-symbol.slice">
+    <h1>Symbol.slice</h1>
+    <p>The initial value of `Symbol.slice` is the well known symbol @@slice (<emu-xref href="#table-1"></emu-xref>).</p>
+    <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+  </emu-clause></ins>
+</emu-clause>
+
+<emu-clause id="sec-properties-of-the-array-prototype-object">
+  <h1>Properties of the Array Prototype</h1>
+  <emu-clause id="sec-array.prototype-@@slice">
+    <h1>Array.prototype [ @@slice ] (start, end)</h1>
+    <p> The initial value of the @@slice property is the same function object as the initial value of the `Array.prototype.slice` property. </p>
+  </emu-clause>
+</emu-clause>
+
+<emu-clause id="sec-properties-of-the-%typedarrayprototype%-object">
+  <h1>Properties of the %TypedArray.prototype% Object</h1>
+  <emu-clause id="sec-properties-of-the-%typedarrayprototype%-@@slice">
+    <h1>%TypedArray%.prototype [ @@slice ] (start, end)</h1>
+    <p> The initial value of the @@slice property is the same function object as the initial value of the `%TypedArray%.prototype.slice` property. </p>
+  </emu-clause>
+</emu-clause>
+
+<emu-clause id="sec-properties-of-the-arraybuffer-prototype-object">
+  <h1>Properties of the ArrayBuffer Prototype Object</h1>
+  <emu-clause id="sec-properties-of-the-arraybuffer-prototype-@@slice">
+    <h1>ArrayBuffer.prototype [ @@slice ] (start, end)</h1>
+    <p> The initial value of the @@slice property is the same function object as the initial value of the `ArrayBuffer.prototype.slice` property. </p>
+  </emu-clause>
+</emu-clause>
+
+<emu-clause id="sec-properties-of-the-sharedarraybuffer-prototype-object">
+  <h1>Properties of the SharedArrayBuffer Prototype Object</h1>
+  <emu-clause id="sec-properties-of-the-sharedarraybuffer-prototype-@@slice">
+    <h1>SharedArrayBuffer.prototype [ @@slice ] (start, end)</h1>
+    <p> The initial value of the @@slice property is the same function object as the initial value of the `SharedArrayBuffer.prototype.slice` property. </p>
+  </emu-clause>
+</emu-clause>
+
+<emu-clause id="sec-left-hand-side-expressions">
+  <h1>Left-Hand-Side Expressions</h1>
+  <h2>Syntax</h2>
+  <emu-grammar>
+    MemberExpression[Yield, Await] :
+      PrimaryExpression[?Yield, ?Await]
+      MemberExpression[?Yield, ?Await] `[` Expression[+In, ?Yield, ?Await] `]`
+      MemberExpression[?Yield, ?Await] `.` IdentifierName
+      MemberExpression[?Yield, ?Await] TemplateLiteral[?Yield, ?Await]
+      SuperProperty[?Yield, ?Await]
+      MetaProperty
+      `new` MemberExpression[?Yield, ?Await] Arguments[?Yield, ?Await]
+      <ins>MemberExpression[?Yield, ?Await] `[` Expression[+In, ?Yield, ?Await]? `:` Expression[+In, ?Yield, ?Await]? `]` </ins>
```

Comment:
```suggestion
      <ins>MemberExpression[?Yield, ?Await] `[` AssignmentExpression[+In, ?Yield, ?Await]? `:` AssignmentExpression[+In, ?Yield, ?Await]? `]` </ins>
```

Reply:
Agreed. `,` would be very confusing here, and if we ever *did* consider adding something like a `x:y` range literal, using `Expression` here would preclude us from adding it, as it would change the behavior of `a[x,y:z]` from `a[(x,y):z]` to `a[x,(y:z)]`. It is better to be more restrictive